default.yml

```diff
@@ -105,6 +123,21 @@ RSpec/LetSetup:
 RSpec/NestedGroups:
   Enabled: false
 
+RSpec/ImplicitBlockExpectation:
+  Enabled: false
+
+RSpec/MessageChain:
+  Enabled: false
+
+RSpec/EmptyExampleGroup:
+  Enabled: false
```

Comment:
is there a reason this should be allowed? seems odd to have empty example groups unless they were just placeholders or something - easy enough to add a simple expectation

Reply:
We do have some empty spec files that were generated and have no specs. I'm willing to let this one go though... I do agree it makes sense that most spec files could probably have at least *one* spec written for them.